python/hail/docs/tutorials/05-tables.ipynb

```diff
@@ -36,7 +36,9 @@
     "\n",
     "You can read a dataset with [hl.read_table](https://hail.is/docs/devel/methods/impex.html#hail.methods.read_table).  It take a path and returns a `Table`.  `ht` stands for Hail Table.\n",
     "\n",
-    "We've prepared a dataset of movie ratings in the Hail native format.  Let's read it!"
+    "We've provided a method to download and import a dataset of moving ratings in the Hail native format. Let's read it!",
```

Comment:
Typo: moving -> movie 

I think you should also add a link to their website where `movie ratings` is.